samples/sensor/veml6031/prj.conf

```
@@ -0,0 +1,8 @@
+CONFIG_PRINTK=y
+CONFIG_GPIO=y
+CONFIG_SHELL=y
+CONFIG_I2C=y
+CONFIG_I2C_STM32=y
+CONFIG_SENSOR=y
+CONFIG_SENSOR_SHELL=y
```

Most of these Kconfigs aren't needed.
```suggestion
CONFIG_SENSOR=y
```

---

Thanks for the reviews.